### DISCUSSION TITLE
1.18.2启动失败 #3370

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#3373] [讨论]
#3370

**Comment by mc-124** [comment]
PCL2可以出个补全mojang JAVA的文件

**Reply by Ad-closeNN** [reply]
~~java不是自己下自己解压的吗...~~
还有就是，市面上的java很多，比如zulu、dragon well、openjdk、网易的java（没错这也能）还有等等等等
如果这样补的话是不是工程量就大了...

**Reply by wuliaodexiaoluo** [reply]
关键是Mojang真的是这么干的
![图片](https://github.com/Hex-Dragon/PCL2/assets/160083863/fa0bbe0f-7365-4abd-aa79-e70f984f7e75)
全是type:file

**Reply by wuliaodexiaoluo** [reply]
要不我发个Issue让龙猫用高校源

**Reply by Ad-closeNN** [reply]
会增加下载时间的吧
~~再说龙猫可能不会理（只是可能）~~

**Reply by wuliaodexiaoluo** [reply]
总好过境外服务器不稳定吧。。。

**Reply by wuliaodexiaoluo** [reply]
> 会增加下载时间的吧 ~再说龙猫可能不会理（只是可能）~



> 总好过境外服务器不稳定吧。。。


感觉会被挪到新功能。。。

**Reply by Ad-closeNN** [reply]
> 要不我发个Issue让龙猫用高校源

哪里有高校源

**Reply by wuliaodexiaoluo** [reply]
清华大学开源软件镜像站有个Adoptium项目
里面有预编译好的Java

**Reply by Ad-closeNN** [reply]
~~好家伙下得挺快, 甚至是直链~~
`https://mirrors.tuna.tsinghua.edu.cn/Adoptium/21/jre/x64/windows/OpenJDK21U-jre_x64_windows_hotspot_21.0.2_13.zip`
可以试试提交一下issue，但估计需要投票

**Reply by mc-124** [reply]
猫曰：别说粗

**Comment by mc-124** [comment]
应该就是java缺文件了

**Comment by wuliaodexiaoluo** [comment]
![图片](https://github.com/Hex-Dragon/PCL2/assets/160083863/dbf8aa3d-eada-464e-b374-8c26d98eb96b)

好几个 PCL.ModNet.NetFile.Thread(NetThread Info)
应该是网络波动没下全
刚才跳闸了，屏幕灭了吓我一跳
